### ISSUE TITLE
Documentation for member validation failure cases

### ISSUE THREAD

**Issue by nzphoenix** [BetterBrief/silverstripe-opauth#37]
Hi,

Have been working with this module and the one area that seems to be a bit tricky is handling if the member is not valid and requires additional details before registration.

It appears when `profilecompletion` is rendered on the Page_Controller, but init() is not called so no requirements are included?

Furthermore whenever I use OpauthRegisterForm it doesn't pass any of the partial data from `populateFromSources` through to it - all fields are blank?

**Comment by dhensby** [comment]
DISCLAIMER: This module depends on [Opauth](https://github.com/opauth/opauth/) which is no longer maintained - we don't recommend continuing to use this module as we aren't actively developing it any longer.

---

> Have been working with this module and the one area that seems to be a bit tricky is handling if the member is not valid and requires additional details before registration.

This is exactly what we've used this module for. You can't register in one step, though - you have to use the OPAuthIdentity and link it to a Member after they complete the second step of registration.

Basically, what we did was use the OPauth response to pre-populate a registration form and allowed the user to fill in the rest.

>It appears when `profilecompletion` is rendered on the `Page_Controller`, but `init()` is not called so no requirements are included?

Interesting, this should be easily patched with a PR - can you oblige?

> Furthermore whenever I use `OpauthRegisterForm` it doesn't pass any of the partial data from `populateFromSources` through to it - all fields are blank?

Have you been able to investigate why this is? Is it a validator issue?

**Comment by nzphoenix** [comment]
Thanks @dhensby - looks like the populateFromSources was working fine, just a misconfiguration on my end.

I've put through a pull request for profilecompletion page so that the base base will load requirements from the standard Page class - similar to the Security class.

**Comment by dhensby** [comment]
OK great, thanks
